test/extended/networking/internal_ports.go

```diff
@@ -186,15 +210,17 @@ func parallelTest(workers int, fns []func() error) []error {
 }
 
 func testRemoteConnectivityCommand(protocol v1.Protocol, localHostPort, host string, port int, echoMessage string) ([]string, error) {
-	var protocolType string
-	var dialCommand string
+	var protocolType, dialCommand, retries string
+
 	switch protocol {
 	case v1.ProtocolTCP:
 		protocolType = "http"
 		dialCommand = fmt.Sprintf("echo?msg=%s", echoMessage)
+		retries = "1"
 	case v1.ProtocolUDP:
 		protocolType = "udp"
 		dialCommand = fmt.Sprintf("echo%%20%s", echoMessage)
+		retries = strconv.Itoa(UDPRetries)
```

Comment:
Don't use constants in these scenarios - a test command should take retries as a parameter.